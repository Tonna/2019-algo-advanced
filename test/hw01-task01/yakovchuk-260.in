30
1 1 1 3 1 5 1 4 6 1 4 9 12 1 10 7 1 3 8 17 19 19 19 20 17 11 26 20 26
7
13 6
24 7
24 6
1 4
19 1
8 10
21 3
