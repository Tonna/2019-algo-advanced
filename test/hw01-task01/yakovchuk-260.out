19
33
30
4
3
24
13
